metrics/integration.go

```diff
@@ -37,7 +37,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds",
 			Help:    "Time duration from the moment the snapshot resource was created till a integration pipelineRun is started in a static environment",
-			Buckets: []float64{0.05, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 15, 30},
+			Buckets: []float64{0.05, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 15, 30, 45, 60},
```

Comment:
This is SLI 2A; 2B is for ephemeral envs

Also for ephemeral envs, we should include numbers up to limit  https://github.com/redhat-appstudio/integration-service/blob/c199d2673bc5c7dc9936798acce1ffd9bad6864b/controllers/binding/snapshotenvironmentbinding_adapter.go#L43 or generaly make gaps between numbers bigger, I don't think that we can cound with deploments under 1s at all

Reply:
Line 48 should be updated in this PR

Reply:
Ack, I'll also remove the first few buckets